src/classes/Ingredient.js

```diff
@@ -1,15 +1,13 @@
-import ingredientsData from '../data/ingredients.js';
-
 class Ingredient {
-  constructor(id, quantity) {
+  constructor(id, quantity, ingredientsData) {
```

Comment:
It's good that we are now taking in the ingredientsData as an argument when invoking the Ingredient class and importing through the scripts.js file, rather than importing the indgredients data directly into the Ingredient class file.